tests/unittests/tests-libcose/Makefile.include

```diff
@@ -0,0 +1,5 @@
+﻿CFLAGS +=-DCOSE_SIGNATURES_MAX=2
+
+USEMODULE += random
+USEPKG += libcose
+USEMODULE += libcose_crypt_tweetnacl
```

Comment:
Why this is duplicated in the Makefile too? Plus, I don't see where this Makefile gets included...

Reply:
Removed it from the `tests/unittests/tests-libcose/Makefile`. This file is included [here](https://github.com/RIOT-OS/RIOT/blob/master/tests/unittests/Makefile#L225)